provider/resource.go

```diff
@@ -2092,10 +2014,14 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_management_group_template_deployment":      {Tok: azureResource(azureManagement, "GroupTemplateDeployment")},
 			"azurerm_management_group_subscription_association": {Tok: azureResource(azureManagement, "GroupSubscriptionAssociation")},
 			"azurerm_management_group_policy_assignment":        {Tok: azureResource(azureManagement, "GroupPolicyAssignment")},
+			"azurerm_management_group_policy_remediation":       {Tok: azureResource(azureManagement, "GroupPolicyRemediation")},
 
 			// communication
 			"azurerm_communication_service": {Tok: azureResource(azureCommunication, "Service")},
 
+			// connections
+			"azurerm_api_connection": {Tok: azureResource(azureConnections, "Connection")},
```

Comment:
I think this should be `ApiConnection`? If intentional, might be good to add a comment why it's not `ApiConnection`.